modules/crafted-ide-config.el

```diff
@@ -48,27 +48,32 @@ manually with something like this:
 (customize-set-variable 'eglot-autoshutdown t)
 
 ;;; tree-sitter
-;; Emacs versions prior to 29
-(when (version< emacs-version "29")
-  (when (require 'tree-sitter-indent nil :noerror)
+(defun crafted-ide--configure-tree-sitter-pre-29 ()
+  "Configure tree-sitter for Emacs 28 or earlier."
+  (when (version< emacs-version "29")
+    (when (require 'tree-sitter-indent nil :noerror)
```

Comment:
Why is this here?  The code which follows doesn't seem to need `tree-sitter-indent` unless I'm missing something - which is possible.

Reply:
Well, the version guard I just forgot to remove (as it's handled by `crafted-ide-configure-tree-sitter`).
As for the require guard, that was already there in the previous version.
I'll remove them both.

Reply:
D'oh!  Probably my fault then.  I wonder why I did that.  Thanks for fixing it!